### PR TITLE
Add reference machine weights

### LINKED ISSUE #377: [Benchmark] Benchmark weights on collator machines
To ensure that our collator nodes are able to produce a block in-time, we have to benchmark every pallet (including external pallets) on one of the collator machines.

### PR DESCRIPTION
**Pull request by sea212**
depends on #383
closes #377

### CODE REVIEW

**Comment by sea212** [comment]
Blocked until #383 is merged.

**Review by lsaether — APPROVED** [review]

**Pull request merged**
